EasyLearn/app/views/shared/home/_cards_classes.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="cards-container">
+  <div class="card-deck">
+    
+    <div class="card">
+      <iframe src="https://www.youtube.com/embed/MfeOZooNamI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <div class="card-body">
+        <h5 class="card-title">Implemented AJAX with PHP</h5>
+        <p class="card-text">Learn how to share information from AJAX to PHP and viceversa.</p>
+        <%= link_to "Take calss", "#", class: "btn" %>
```

Comment:
It should be like that `<%= link_to "Take class", "#", class: "btn" %>`, no?

Reply:
Done.